Chp8Lsn4/Chp8Lsn4/Program.cs

```diff
@@ -0,0 +1,110 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Chp8Lsn4
+{
+    public class Program
+    {
+        static void Main(string[] args)
+        {
+            int decimalNumber = UserInput();
```

Comment:
Remember "self documenting code".  Why is this variable called "decimalNumber", when it contains an integer?  For that matter, the name of the variable does not need to include its type.  For example:

 * A variable called NameString could just be Name
 * A variable called intAge could just be called Age

Programmers are not forbidden from saying the name of the type as part of the variable, but it should probably be uncommon or rare.